### ISSUE TITLE
Table thead with td instead of th

### ISSUE THREAD

**Issue by janis-veinbergs** [mysticmind/reversemarkdown-net#57]
Happens when table gets copied out of MS Word & converted to HTML.
It interprets first row within `thead` as columns - good. But also interprets `tbody` first row as columns.

Given html:
```html
<table><thead><tr><td>col1</td><td>col2</td></tr></thead><tbody><tr><td>data1</td><td>data2</td></tr><tbody></table>
```

Expected: 
```
| col1 | col2 |
| --- | --- |
| data1 | data2 |
```

Actual:   
```
| col1 | col2 |
| --- | --- |
| data1 | data2 |
| --- | --- |
```

**Comment by mysticmind** [comment]
Fixed via #58

**Comment by mysticmind** [comment]
@janis-veinbergs v3.9.0 released to NuGet.
